components/meter/meter-linear.js

```diff
@@ -75,25 +76,20 @@ class MeterLinear extends LitElement  {
 	}
 
 	render() {
-		const primaryTextId = getUniqueId();
-		const secondaryTextId = getUniqueId();
-
 		const percentage = this.max > 0 ? this.value / this.max * 100 : 0;
 		const primary = this.percent ? `${Math.floor(percentage)}%` : `${this.value}/${this.max}`;
 		const secondaryText = this._formatContext(this.text, this.value, this.max);
-		const secondaryTextElement = secondaryText ? html`<div id="${secondaryTextId}" class="d2l-meter-linear-secondary">${secondaryText}</div>` : html``;
+		const addSpace = !this.textInline && secondaryText !== this.text ? 'd2l-meter-linear-text-space-between' : '';
```

Comment:
Using the [classMap](https://lit-html.polymer-project.org/guide/template-reference#classmap) stuff might be a bit cleaner here?